clang-tools-extra/test/clang-tidy/checkers/modernize/loop-convert-basic.cpp

```diff
@@ -1,4 +1,6 @@
-// RUN: %check_clang_tidy %s modernize-loop-convert %t -- -- -I %S/Inputs/loop-convert
+// RUN: %check_clang_tidy %s modernize-loop-convert %t -- -- -I %S/Inputs/loop-convert -isystem %clang_tidy_headers
+
+#include <cstddef>
```

Comment:
you could just use `unsigned int size` in line 94 instead of including this header.